tutorials/plot_dipole_orientations.py

```diff
@@ -0,0 +1,196 @@
+"""
```

Comment:
I suppose there's no difference. We seem to do ``# -*- coding: utf-8 -*-`` elsewhere.